rpm/BUILD.bazel

```diff
@@ -72,7 +73,7 @@ rpmtree(
         "@glib2-0__2.56.4-156.el8.aarch64//rpm",
         "@glibc-0__2.28-164.el8.aarch64//rpm",
         "@glibc-common-0__2.28-164.el8.aarch64//rpm",
-        "@glibc-langpack-bho-0__2.28-164.el8.aarch64//rpm",
+        "@glibc-minimal-langpack-0__2.28-164.el8.aarch64//rpm",
```

Comment:
This, as well as the acl bit above, are unrelated to the addition of Python and should appear in the corresponding commits instead.

Reply:
I have now accumulated all WORKSPACE updates and all RPM file updates to one final commit.